src/common/LoggedInProviders.tsx

```diff
@@ -49,21 +49,23 @@ export const LoggedInProviders = ({ children }: LoggedInProvidersProps) => {
   }
 
   return (
-    <ActiveProjectContextProvider>
-      <TrackTileProvider>
-        <WearableLifecycleProvider>
-          <CircleTileContextProvider>
-            <OnboardingCourseContextProvider>
-              <PushNotificationsProvider config={pushNotificationsConfig}>
-                <NotificationsManagerProvider>
-                  {children}
-                </NotificationsManagerProvider>
-              </PushNotificationsProvider>
-              <CreateEditPostModal />
-            </OnboardingCourseContextProvider>
-          </CircleTileContextProvider>
-        </WearableLifecycleProvider>
-      </TrackTileProvider>
-    </ActiveProjectContextProvider>
+    <InviteProvider>
```

Comment:
I think I'm missing something: conceptually The `LoggedInProviders` component seems to indicate that the user is already logged in, so at that point an invite has already been accepted. Doesn't the `InviteProvider` play a role before we have a logged in user?

Reply:
> at that point an invite has already been accepted

This is not correct. The typical flow for a brand new user is:

1. Log in via `evc` + the OpenID issuer.

2. Once logged in, accept the invite (this is an authenticated API call)

3. Once invite is accepted, show consent + rest of app.

Reply:
@aecorredor So, the role of the `InviteProvider` is to accomplish step 2.

Reply:
Thanks for the clarification.